query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -51,7 +51,7 @@ impl ToSqlRow for ResultRow {
                         return Err(SqlError::ConversionError(error.into()));
                     }
                 },
-                (type_identifier, _) => row_value_to_prisma_value(p_value, type_identifier),
+                (type_identifier, _) => row_value_to_prisma_value(p_value, type_identifier.clone()),
```

Comment:
Same as above.

Reply:
That was already addressed after the review with @tomhoule